quickstart/deploy-mayastor.md

```diff
@@ -75,15 +75,26 @@ kubectl -n mayastor get pods --selector=app=nats
 
 {% tab title="Example Output" %}
 ```text
-NAME          READY   STATUS    RESTARTS   AGE
-nats-0        3/3     Running   0          61s
+NAME     READY   STATUS    RESTARTS   AGE
+nats-0   2/2     Running   0          50s
+nats-1   2/2     Running   0          30s
+nats-2   1/2     Running   0          10s
```

Comment:
The nats-2 should be having 2/2 containers in ready state.

Reply:
Fixed under PR #45